packages/components/src/Modal/Modal.types.ts

```diff
@@ -30,6 +30,16 @@ export interface ModalProviderProps {
    */
   readonly activatorRef?: MutableRefObject<HTMLElement | null> | null;
   readonly dismissible?: boolean;
+  /**
+   * Id to provide aria-labelledby to the modal. If you are using Modal.Header with children or another header you will need to provide this as the id of the header.
+   * @default "ATL-Modal-Header"
+   */
+  readonly modalLabelledBy?: string;
```

Comment:
I'm confused. why do we need to add this? this already exists and works no?

Reply:
removed `ModalProviderProps` from the types file here 073b526ffbe92c729f8d0ecff79cc5a2a2a11114

I can make a follow up task to move types and export properly